bold-linkage/main.py

```diff
@@ -213,12 +177,13 @@ def run_local(example: str) -> None:
     """
     response = requests.get(example)
     specimen = json.loads(response.content)["data"]
-    specimen_data = specimen["attributes"]["digitalSpecimen"]
+    specimen_data = specimen["attributes"]
     result = run_api_call(specimen_data)
-    mas_job_record = map_to_mas_job_record(specimen_data, result, str(uuid.uuid4()))
+    mas_job_record = map_to_annotation_event(specimen_data, result,
+                                             str(uuid.uuid4()))
     logging.info("Created annotations: " + json.dumps(mas_job_record, indent=2))
 
 
 if __name__ == "__main__":
     start_kafka()
-    # run_local("https://sandbox.dissco.tech/api/v1/specimens/SANDBOX/NMT-F9R-FWK")
+     #run_local("https://sandbox.dissco.tech/api/v1/specimens/SANDBOX/NMT-F9R-FWK")
```

Comment:
Test code that can be removed?

Reply:
We've been leaving the `run_local` functions here for convenience, i think it would be useful for MAS developers to have a local example too

Reply:
Maybe add a comment explaining that, or is it in the description?